components/sst_importer/src/errors.rs

```diff
@@ -60,6 +61,9 @@ quick_error! {
             display("Invalid SST path {:?}", path)
         }
         InvalidChunk {}
+        RocksEngine(err: Box<dyn StdError + Send + Sync + 'static>) {
```

Comment:
I didn't realize this variant was inside sst_importer when I made it. I'll rename it to not include 'rocks'.